content/docs/start/experiments/experiment-tracking.md

```diff
@@ -117,7 +117,10 @@ with Live(save_dvc_exp=True) as live:
 </toggle>
 
 After this, each execution of the code will create a <abbr>DVC experiment</abbr>
-containing the results and the changes needed to reproduce it.
+containing the results and the changes needed to reproduce it. Each experiment
+is
+[automatically saved in Git](/doc/user-guide/experiment-management#how-does-dvc-track-experiments),
+so you don't need to worry about committing them.
```

Comment:
```suggestion
so you don't need to worry about committing them. Note that these are not pushed to Git remotes by default.
```

Adding the sentence so that the user does not interpret `automatically saved in Git` as `it'll be there in my Git remote now`.